whisper_medusa/dataset/dataset.py

```diff
@@ -0,0 +1,171 @@
+import logging
+from dataclasses import dataclass
+from typing import Dict, List, Union
+import pandas as pd
+import torch
+import torchaudio
+from transformers import WhisperProcessor
+from whisper_medusa.utils.utils import token_padding
+
+SAMPLE_RATE = 16_000
+
+
+class ASRDataSet(torch.utils.data.Dataset):
+    """
+    ASR Dataset
+
+    Parameters
+    ----------
+    data_path : str. Path to csv file with columns: "sentence", "path".
+    """
+
```

Comment:
update docstring to include all input args.

Reply:
done